billboard.py

```diff
@@ -206,7 +207,16 @@ def getRowValue(rowName):
                 message = "Failed to parse rank"
                 raise BillboardParseException(message)
 
-            entry = ChartEntry(title, artist, peakPos, lastPos, weeks, rank)
+            try:
+                newEntry = False
+                selector = 'div.chart-row__new-indicator'
+                if entrySoup.select_one(selector):
+                    newEntry = True 
+            except BillboardParseException:
```

Comment:
This should just be `except:`. There's nothing in the try-block that can raise a `BillboardParseException`. In the other parts of the code, I catch `BillboardParseException` because `getRowValue` raises it.